data/mods/Aftershock/player/bionics.json

```
@@ -63,6 +63,86 @@
     "react_cost": "12 kJ",
     "time": 1
   },
+  {
+    "id": "bio_forcefield_bash_cut_weak",
+    "type": "bionic",
+    "name": { "str": "Weak Gravity Manipulation Field" },
+    "description": "Imagine a computer inside your body playing pong against the universe.  The paddles are made of gravity fields and how it causes this to happen are trade secrets.  -From the owners manual of HawkingCorps GravyField Mark I.",
+    "occupied_bodyparts": [ [ "torso", 8 ] ],
+    "available_upgrades": [ [ "bio_forcefield_bash_cut_medium" ], [ "bio_forcefield_bash_cut_heavy" ] ],
+    "act_cost": "1 kJ",
+    "react_cost": "1 kJ",
+    "time": 1,
+    "enchantments": [ "forcefield_bash_cut_weak" ],
+    "flags": [ "BIONIC_TOGGLED" ]
+  },
+  {
+    "id": "bio_forcefield_bash_cut_medium",
+    "type": "bionic",
+    "name": { "str": "Combat Rated Gravity Manipulation Field" },
+    "description": "The HawkingCorps GravyField Mark 2 was designed for police actions during food riots and other civilian uprisings.  It has proven cabaple of protecting against most improvised weapons.",
+    "occupied_bodyparts": [ [ "torso", 10 ] ],
+    "upgraded_bionic": "bio_forcefield_bash_cut_weak",
+    "available_upgrades": "bio_forcefield_bash_cut_heavy",
+    "act_cost": "5 kJ",
+    "react_cost": "5 kJ",
+    "time": 1,
+    "enchantments": [ "forcefield_bash_cut_medium" ],
+    "flags": [ "BIONIC_TOGGLED" ]
+  },
+  {
+    "id": "bio_forcefield_bash_cut_heavy",
+    "type": "bionic",
+    "name": { "str": "Debt Collection Agent Personal GravyField" },
+    "description": "This class of personal gravity manipulator often sees action of both sides of the vicious ship to ship boarding fights that are common with both repossession and piracy.  It remains practically impenetrable to melee weapons that lack sharp piercing points.",
```

```suggestion
    "description": "This class of personal gravity manipulator often sees action on both sides of the vicious ship-to-ship boarding fights that are common with both repossession and piracy.  It remains practically impenetrable to melee weapons that lack sharp piercing points.",
```
Why would piercing points make a difference for gravity manipulation?

---

I'm open to suggestions for a different type of forcefield that is strong vs bash/cut and no protection against stab.